_spinnaker_user_guides/plugin-creators.md

```diff
@@ -153,78 +153,111 @@ public class RandomWait<RandomWaitInput> {
-
-## Writing The Frontend
+### Rollup Configuration
+Here is an example of a `rollup.config.js` to build your plugin:
 
 ```
-import * as React from 'react';
-// IPluginInitialize is function interface
-// that takes in the IStageRegistry interface.
-// The IStageRegistry is used to register the stage.
-import { IPluginInitialize, IStageRegistry } from '@spinnaker/plugins';
-
-// Our stage component
-class RandomWaitStage extends React.Component {
-  setMaxWaitTime = (event: React.SyntheticEvent) => {
-    let target = event.target as HTMLInputElement;
-    // @ts-ignore
-    this.props.updateStageField({'maxWaitTime': target.value});
+import nodeResolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import typescript from 'rollup-plugin-typescript';
+import postCss from 'rollup-plugin-postcss';
+import externalGlobals from 'rollup-plugin-external-globals';
+
+export default [
+  {
+    input: 'src/index.tsx',
+    plugins: [
+      nodeResolve(),
+      commonjs(),
+      typescript(),
+      // Map imports from shared libraries (React, etc) to global variables exposed by Spinnaker.
+      externalGlobals(spinnakerSharedLibraries()),
+      // Import from .css, .less, and inject into the document <head></head>.
+      postCss(),
+    ],
+    output: [{ dir: 'dist', format: 'es', }]
   }
+];
+
+function spinnakerSharedLibraries() {
+  const libraries = ['react', 'react-dom', '@spinnaker/core'];
 
-  render() {
-    return (
-      <div>
-        <label>
-            Max Time To Wait
-            <input onChange={this.setMaxWaitTime} id="maxWaitTime" />
-        </label>
-      </div>
-    );
+  function getGlobalVariable(libraryName) {
+    const prefix = 'spinnaker.plugins.sharedLibraries';
+    const sanitizedLibraryName = libraryName.replace(/[^a-zA-Z0-9_]/g, '_');
+    return `${prefix}.${sanitizedLibraryName}`;
   }
+
+  return libraries.reduce((globalsMap, libraryName) => {
+    return { ...globalsMap, [ libraryName ]: getGlobalVariable(libraryName) }
+  }, {});
 }
+```
 
-// This function implements the IPluginInitialize interface
-// This is where the stage gets registered.
-function initialize(registry: IStageRegistry): void {
-  registry.pipeline.registerStage({
-    label: 'Random Wait',
-    description: 'Stage that waits a random amount of time up to the max inputted',
-    key: 'randomWait',
-    component: RandomWaitStage,
-  });
-};
+`spinnakerSharedLibraries` is used to pull depenedencies from Spinnaker. To see the list of depenedcies Spinnaker exposes to plugin creators, please take a look at [shared libraries](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/plugins/sharedLibraries.ts#L32). `libraries` is the list of libraries needed from Spinnaker to make a plugin work correctly. This list must contain libraries from the shared libraries section in Spinnaker.
```

Comment:
Best to use active voice, simple and direct language, and avoid unnecessary phrases like _please_.

Suggestion:
`spinnakerSharedLibraries` pulls dependencies from Spinnaker. The `libraries` constant is a list of libraries that make the plugin work correctly. Items in this list must be from the [shared libraries](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/plugins/sharedLibraries.ts#L32) exposed to plugin creators.